library_management/src/main/java/com/group2/library_management/entity/RoleType.java

```diff
@@ -0,0 +1,32 @@
+package com.group2.library_management.entity;
```

Comment:
Suggest đưa vào package enum nhé 
```suggestion
package com.group2.library_management.entity.enums;
```